packages/stryker/src/StrykerCli.ts

```diff
@@ -47,6 +47,7 @@ export default class StrykerCli {
       .option('--timeoutFactor <number>', 'Tweak the standard deviation relative to the normal test run of a mutated test', parseFloat)
       .option('--maxConcurrentTestRunners <n>', 'Set the number of max concurrent test runner to spawn (default: cpuCount)', parseInt)
       .option('--logLevel <level>', 'Set the log4js log level. Possible values: fatal, error, warn, info, debug, trace, all and off. Default is "info"')
+      .option('--tempFolder <name>', 'Set name of the folder that contains temporary files')
```

Comment:
Should we call it `tempFolder` or `tempDir` (or `tempDirectory`). I think `tempDir` is more clear, we should always refer to `directory` instead of `folder`.

Reply:
I think `tempDir` is better. But if we change it we should also change the `TempFolder` class.

Reply:
Renamed to tempDir